frontend/components/organisms/AnswerComponent/index.tsx

```diff
@@ -1,27 +1,80 @@
 import React from "react";
 import RichTextEditor from "../../molecules/RichTextEditor";
 import Button from "../../atoms/Button";
+import { useForm } from "react-hook-form";
+import { useMutation } from '@apollo/client';
+import toast, { Toaster } from 'react-hot-toast';
+import CREATE_ANSWER from "../../../helpers/graphql/mutations/create_answer";
+
+export type FormValues = {
+  content: string;
+  user_id: number;
+  question_id: number;
+};
+
 
 const AnswerComponent = () => {
-  const handleOnClick = () => {
-    console.log("clicked");
-  };
+  const [createAnswer] = useMutation(CREATE_ANSWER);
+
+  const { setValue, handleSubmit } = useForm<FormValues>({
+    mode: "onSubmit",
+    reValidateMode: "onSubmit"
+  });
+
+
+  const onSubmit = (data: FormValues) => {
+    data.question_id = 1
+    data.user_id = 1
+
+    if (data.content !== undefined) {
+      if (data.content.replace(/<(.|\n)*?>/g, '').trim().length === 0) {
+        toast.error('No answer Input');
+      }
+      else {
+        const new_answer = createAnswer({
+          variables: {
+            content: data.content,
+            user_id: data.question_id,
+            question_id: data.user_id
+          }
+        })
+
+        toast.promise(new_answer, {
+          loading: 'Adding Answer...',
+          success: 'Answer Successfully Added!',
+          error: 'There was an Error!',
+        });

```

Comment:
You can add a plugin file where you have an export function for success, error, info toast

Reply:
```suggestion
// example plugin/helper file 
import toast, { Toaster } from 'react-hot-toast';

export function formProcessToast(promise, loadingMessage = "Processing ...", successMessage = "Process done", errorMessage = "Error in process",position = "top-right") {
    return toast.promise(promise, {
        loading: loadingMessage,
        success: successMessage,
        error: errorMessage,
    });
}
```

Noticed that it does not have position prop maybe you can try [React toastify](https://www.npmjs.com/package/react-toastify)